schemas/categories.txt

```diff
@@ -0,0 +1,67 @@
+- Barrel Scanner
+A fixture which has a rotating cylinder of mirrors to reflect the light in a moving pattern.
+Example: american-dj/inno-pocket-fusion = https://www.adj.com/inno-pocket-fusion
+
+- Blinder
+Extremely bright fixtures intended to light the audience, rather than the stage.
+Example: elation/cuepix-blinder-ww2 = https://www.elationlighting.com/cuepix-blinder-ww2
+
+- Color Changer
+A fixture which can change its color electronically, especially (but not only) one with a gel wheel.  This includes RGB LEDs.
+Example: clay-paky/sharpy = https://www.claypaky.it/en/products/sharpy
```

Comment:
Fixtures with a color wheel should also be included in the "Color Changer" category. Basically, it's every fixture with the ability to change the color of its light output.

Reply:
That's what I was trying to say, but it wasn't as clear as I'd hoped.  How about something like:

"A fixture which can electronically change its color, like with a wheel of gels, or RGB LEDs.  (A fixture which can only modulate warm/cool tints of white is not considered a color changer.)"

For some reason I'm having trouble making this sound not-awkward.